example/macos/Flutter/flutter_config.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+# This is a generated file; do not edit or check into version control.
+
+touch Flutter/ephemeral/tripwire
```

Comment:
This script should probably go somewhere else

Reply:
Why is it a separate script and not part of the script step that calls macos_assemble? Or even better, internal to macos_assemble so we can change it later without any project changes?

Reply:
Ahh duh, that is much easier :)